### PR TITLE
Added inverse string equal to file support.

### PR DESCRIPTION
**Pull request by milesj**
Finally got around to this. Implements #8.

### CODE REVIEW

File: test/file-test.js [hunk 1]
```diff
@@ -383,3 +381,70 @@ describe('expect(file(...))', function() {
...
+    describe(methodDesc + '(file(...))', function() {
+      it('passes for string to equal file', function() {
+        expect('This is a small fixture file called "foo.txt"!\n').to[method](file('test/fixtures/foo.txt'));
+      });
+
+      it('fails for string not equal to file', function() {
+        expect(function() {
+          expect('large solid object').to[method](file('test/fixtures/foo.txt'));
+        }).to.throw(function(err) {
+          expect(err.toString()).to.equal('AssertionError: expected \"test/fixtures/foo.txt\" to equal \"large solid object\"');
```
Comment (Turbo87) [inline test/file-test.js]: The message is more or less reversed here. Maybe we could add an `inverse` parameter to the `assertEqual()` method to flip it?

Comment (milesj) [inline test/file-test.js]: Sure can do.

**Comment by milesj** [comment]
Updated.

**Comment by Turbo87** [comment]
awesome, couldn't have done it better 👍

**Comment by milesj** [comment]
Cheers!

**Comment by milesj** [comment]
Could you publish a new version that includes this?

**Comment by Turbo87** [comment]
@milesj just released v1.3.0. thanks again!

**Comment by milesj** [comment]
Awesome thank you!

**Pull request merged**